scripts/log-session.py

```
@@ -0,0 +1,334 @@
+        # Look for task reference patterns
+        match = re.search(r'(?:Task ID|TASK)[:\s]+([A-Z]+-\d+)', content, re.IGNORECASE)
+        return match.group(1) if match else None
+    except Exception:
+        return None
+
+
+def get_modified_files() -> list[str]:
+    """Get list of modified files from git status."""
+    try:
+        result = subprocess.run(
+            ['git', 'diff', '--name-only', 'HEAD~1'],
```

Ruff (0.14.14)

[error] Starting a process with a partial executable path

(S607)